doc/testing_samples/sidewalk_application_samples/template_ble.rst

```diff
@@ -242,8 +246,9 @@ You should see the following output:
 
 Wait for the device to complete the :ref:`automatic_registration_sidewalk_endpoints`.
 
-To test the device, follow the steps in :ref:`sidewalk_testing`.
+.. include:: testing_sample_include.txt
```

Comment:
I added the include with testing instructions to 2 samples: template_ble and template_subghz. Should it be added to the remaining samples as well? Seemed like the testing process for dut and sensor monitoring is different.

Reply:
You're right, sensor monitoring and dut are different, so this testing instructions doesn't apply to them